### ISSUE TITLE
Problem with getContacts() callback

### ISSUE THREAD

**Issue by kunal95** [LynxITDigital/react-native-contacts-wrapper#6]
I am calling this function only once:
```
ContactsWrapper.getContact()
        .then((c) => {
          if(this.state.isEmail) this.emailSelected(c.email);
          else this.contactSelected(c.phone);
        })
        .catch((error) => {
          console.warn("ERROR CODE: "+ error.code);
          alert("Error: "+ error.message);
        });
```

RN 0.28 (Android)

![screenshot_2016-09-17-20-18-59](https://cloud.githubusercontent.com/assets/5383258/18609397/bb6f5416-7d1e-11e6-9e51-86da895eb8bf.png)

**Comment by JaxGit** [comment]
I couldn't reproduce with ContactsWrapper 0.1.3, RN 28/30 on Android 4.2.2 device unless pressing the function triggering button multiple times. 

If that was the case, please keep a flag to prevent ContactsWrapper.getEmail() from getting called again when importing info.

**Comment by kunal95** [comment]
The problem was this module's onActivityResult being fired from super.onActivityResult from MainActivity

Sorted it by putting a not NULL check on this module's onActivityResult

**Comment by brycepavey** [comment]
@kunal95 I came across this just yesterday in a project we're working on. The problem seemed to occur when we were using this in conjunction with another native bridged module (react-native-image-picker), and crashed when an image was selected in the Android image picker, giving a null pointer exception within this contacts wrapper library. 
I've added robustness code to return if the promise is null, or the request code is not one of the ones we use, this should fix your error. 
I've just published v0.2.2 with these changes
